README.md

```diff
@@ -31,6 +31,26 @@ Here is a brief list of the available widgets:
 * **edit_box**: an editable text box (multi-line edit_boxes are not yet fully supported).
 * **scroll_frame**: a frame that has scrollable content.
 
+# Getting started 
+```
+git clone https://github.com/unnamed-mmo/lxgui
+sudo dnf install -y freetype-devel SFML-devel lua-bit32 mesa-libGLU-devel lua-devel
+cd lxgui
+git submodule update --init
+```                                                                       
+## install GLEW                                                                                                    
+download tar of the [![latest release](https://github.com/nigels-com/glew/releases/)](https://github.com/nigels-com/glew/releases/)                                                                                 
+```
+sudo dnf -y install libXmu-devel libXi-devel libGL-devel dos2unix git wget gcc
+```
+unzip, enter the dir and 
+```
+sudo make install 
+```
+# Quickstart 
+
+fork the [![bootstrap project](https://github.com/wgahnagl/lxgui-boilerplate)](https://github.com/wgahnagl/lxgui-boilerplate) and get started right away
```

Comment:
Great to see someone else's project using the library :) However, how is this different from the test app in `lxgui/gui/test` ? If you found it difficult to find and/or use this test application, perhaps we can see how to make it more visible and easier to use, rather than having people clone another repo (which could get out of sync, etc.).

Reply:
for sure! 
The thing I was having the biggest issue with was writing the cmake file to include lxgui as a library instead of building a test within the project. I'm a c++ newbie, so I wasn't sure what I needed to link, and what was specific to the building lxgui library itself and what was specific to building the test.

I tried to keep your test file as intact as possible, so I only made a few really small changes (like outputting the logs to a log file). Also I was getting a weird sol2 error that I solved by deleting a line of code in my fork of it, lol 
I'm pretty sure I shouldn't have done that and it's just my machine pitching a fit, but just in case it isn't, I want people to be able to be able to get going as painlessly as possible.
I made the boilerplate project because of how useful I've found them when I'm hacking stuff together with brand new frameworks. I do a lot of hackathons, so having something that builds an example that I can modify and hack together into my own project can be really nice, but I'm not married to the idea! 
It would probably be way better to include it with this repo!

Reply:
I see your point. It does make sense to write the test/demo/bootstrap as if it was a true third-party rather than rely on it being in the project's CMake hierarchy. I think this should be done.

I am currently working on implementing an SDL backend for lxgui, and I'll have to make substantial edits to this test project as I go. I propose to take out the "bootstrap" aspect of this PR, and open an issue to improve the test project instead. This way, I'll remember to pick it up when I'm getting to it :)

Reply:
can do! I made sure to keep that part in a different commit anyway, just in case lol